Fix a few bugs, and make a few clarifications in the android quick start guide. (#3512)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/createproject.md

> **Note:** The guide requires Android Studio (Arctic Chipmunk 2021.2.1)

1. Create a new Kotlin Android project called `browser_sign_in` with an **Empty Activity** in Android Studio. For information on creating a project, see the [Android Studio documentation](https://developer.android.com/training/basics/firstapp/creating-project).

    - Name: `browser_sign_in`
    - Package name: `com.okta.android.samples.browser_sign_in`
    - Language: `Kotlin`
    - Minimum SDK: `API 23: Android 6.0 (Marshmallow)`

> 2. Add the following code to the `app/build.gradle` to enable Android features used by the Okta SDK that aren't available on older Android versions:

Reviewer: Do these add features that are after Android 6.0? Are they required or optional features, that is, must these packages load for an app to run?

This example assumes the packages are required... it requires the minimum Android version

> 2. Add the following code to the `app/build.gradle` file to enable features used by the Okta SDK that require Android xx.xx or later:

Author: These are required for the app to run.

Reviewer: The important thing is to be specific about what versions of Android: "older Android versions" doesn't tell them. They need to know what versions of Android are supported, which appears to be on line 8. So if this line is loading packages that are Android 6 or later, then the line could be (the fact that it's Android 6.0 or later is taken care of above... if the packages require later versions of Android, then the language needs to change to indicate that.):
```suggestion
2. Add the following to the `app/build.gradle` to enable features required by the Okta Android SDK:
```
